stack/stack_mac.yaml

```diff
@@ -62,8 +62,8 @@ nccmp:
   version: 1.8.9.0
 
 nco:
-  build: YES
-  version: 5.0.6
+  build: NO
+  version: 4.9.3
```

Comment:
Why do you lower the version here? I've had success building with 5.0.6.

Reply:
Higher version could be left in the stack.yaml, as it is not being built on the MacOS. The software has an issue being build during hpc-stack bundled install. Directions for Mac are to install the library via "brew install libpng"

Reply:
We've built 5.0.1 and 5.0.6 on macOS with clang+gfortran as well as gcc+gfortran.

Reply:
Tested building with nco v.5.0.6 - worked OK! Placing it back to the stack.